main.js

```diff
@@ -52,43 +52,41 @@ function compareColorId(event) {
   }
 }
 
-buttonNewPalette.addEventListener("click", refreshPalette);
+buttonNewPalette.addEventListener("click", loadPalette);
 buttonSavedPalette.addEventListener("click", savePalette);
 
 //FUNCTIONS
-function deleteSavedPalette() {
-  console.log("steve", savedPalettes);
-
+function deleteSavedPalette(event) {
+  console.log(event.target.parentElement)
+  var trashID = parseInt(event.target.id)
   for (var i = 0; i < savedPalettes.length; i++) {
```

Comment:
I see we ended up having to use event.target.id. Good to have a visual representation of it actually being used.